src/test/java/com/PlaceLab/Amila/qamp/tests/LoginFunctionalityForgotPassword.java

```diff
@@ -0,0 +1,59 @@
+package com.PlaceLab.Amila.qamp.tests;
+
+import com.PlaceLab.Amila.qamp.utils.WebDriverSetup;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+//TC 05 - Forgot Password Functionality - Verify user can reset their password using the "Forgot Password" feature.
+public class LoginFunctionalityForgotPassword {
+    private WebDriver driver;
+
+    @Parameters("browser")
+    @BeforeTest
+
+    public void setup(String browser) {
+        driver = WebDriverSetup.getWebDriver(browser);
+        driver.get("https://demo.placelab.com");
+        System.out.println("Opened browser: " + browser);
+    }
+
+    @Test
+    @Parameters("password")
+    public void testForgotPassword(String password) {
+
+        Assert.assertTrue(driver.findElement(By.id("login")).isDisplayed(), "Verify if Login page is visible.");
+
+        // Click on the "Forgot Password" link
+        driver.findElement(By.className("link-btn")).click();
+
+        //Verify Forgot Password page is opened
+        Assert.assertTrue(driver.findElement(By.className("span12")).isDisplayed());
```

Comment:
what is the span12 element? can you find the more unique element to locate? I see that the class span12 fetches two elements, we need just one element to be located with our locator